index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="./public/style/main.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Cuprum&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/40e0a47e84.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="./src/lib/setInputStep.js"></script>
+</head>
+
+<body>
+  <div class="outer-container">
+    <div class="container-1">
+      <div>
+        <h2 class="head-title">Colorimetry</h2>
+      </div>
+      <div class="description">
+        <h4 class="head-description">A tool for measuring indoor pollution level, the formaldehyde concentration using
+          colorimetry badge.</h3>
+      </div>
+    </div>
+    <div class=" screen container-2">
+      <div class="box-1">
+        <div class="upper-wrapper">
+          <h2 class="head-screens">1. Let's compare the colored patches.</h3>
+            <div id="drop-area" class="input-img">
+
+              <form class="my-form">
+                <h4 class="selector">
+                  Drag your image into this box or <br><br>
+                </h4>
+                <input type="file" id="fileElem" multiple accept="image/*" onchange="handleFiles(this.files)">
+                <label class="button" for="fileElem" style="border-bottom: solid #727171 2px"><span
```

Comment:
Don't know exactly the coding style of this code base, but I usually try to avoid css inline styles by putting them on a separate file ;)

Reply:
Sure, I will make these changes.
Thanks a lot!